### PR TITLE
feat: control the margin top of the peaks label

### PR DESCRIPTION
**Pull request by hamed-musallam**

### CODE REVIEW

File: src/component/reducer/preferences/panelsPreferencesDefaultValues.ts
Comment (targos) [inline src/component/reducer/preferences/panelsPreferencesDefaultValues.ts:143]: How do you decide it has 3 decimals?

Comment (targos) [reply src/component/reducer/preferences/panelsPreferencesDefaultValues.ts:143]: Ok, then @jobo322 how did you decide? I'm asking since other defaults are either 2 or 5. This is the first occurrence of 3.

Comment (hamed-musallam) [reply src/component/reducer/preferences/panelsPreferencesDefaultValues.ts:143]: As I understand it, the format depends on the default values used in the peak shape, so I assume they used this format based on that.

**Comment by targos** [comment]
Do you need feedback from Luc or we can merge this? I'd like to finish https://github.com/cheminfo/nmrium/pull/3271 this morning

**Comment by hamed-musallam** [comment]
I'm waiting for Luc's feedback before merging (Which I prefer), but if you're in a hurry, we can go ahead and merge it now and make improvements later.

**Review by lpatiny — APPROVED** [review]
Functionality looks good to me.

**Pull request merged**
